$NetBSD: patch-ag,v 1.9 2011/12/15 18:49:53 marino Exp $

All shared libraries need to directly linked to satisfy default behavior of
ld v2.22

--- Imakefile.orig	2005-06-03 12:26:03.000000000 +0000
+++ Imakefile
@@ -102,7 +102,7 @@ SGI_LIB = -limage
 #endif
 
 DEPLIBS = always xpaint.man XPaint.ad
-LOCAL_LIBRARIES = -Lrw -lrw $(XPM_LIB) $(TIFF_LIB) $(SGI_LIB) $(JPEG_LIB) $(PNG_LIB) -ldl -rdynamic
+LOCAL_LIBRARIES = -Lrw -lrw $(XPM_LIB) $(TIFF_LIB) $(SGI_LIB) $(JPEG_LIB) $(PNG_LIB) -lm -lX11 -lXt -lXext
 
 SUBDIRS = rw
 
